content/integrations/dcm-integration.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Dart Code Metrics integration
+description: How to integrate your workflows with Dart Code Metrics using codemagic.yaml
+weight: 18
+---
+
+**Dart Code Metrics** is a powerful static analysis tool that helps improve code quality, ensure code consistency, and identify potential issues early in the development process. 
```

Comment:
You need to add somewhere that you need a Team license key to run on CI/CD. This is where I would link to the pricing page https://dcm.dev/pricing/

Reply:
added to step 1
To effectively integrate Dart Code Metrics (DCM) into your Codemagic CI/CD pipeline, it’s essential to obtain a [Team Plan API key](https://dcm.dev/pricing/).